translations.tsv

```diff
@@ -1,238 +1,238 @@
+hLog20200614	Fixed checkboxes for aurora not updating properly. Restricted Bunny Day special weather to 2020. Added 'possible patterns' display to the seed finder, and warnings for event days. Added 12-hour display option.	Corregido: Las opciones de aurora no se actualizaban correctamente. El patrón para el evento de la Caza del Huevo esta ahora restringido al 2020. Añadido: Muestra de 'patrones posibles' a la pestaña Semilla, advertencias para días con patrones de evento, y opción para mostrar tiempos en el sistema de 12 horas para los americanos.	Correction des checkbox des aurores qui ne se mettaient pas à jour correctement. Restriction du climat spécial du jour du lapin à 2020. Ajout de 'cas possibles' dans la recherche de graine, et avertissements pour les journées évenementielles. Ajout de l'option d'affichage par 12-heures.
+hLog20200615	Added translation support and a Spanish translation.	Añadido: Soporte para traducciones, y una traducción a español.	Ajout du support de traduction et une traduction espagnole
+hLog20200616	Fixed bug involving meteor shower gaps ending in the last minute of the hour (X:59).	Corregido: Un error en que los huecos en el horario de meteoritos no podían acabar en el ultimo minuto de la hora (X:59).	Correction d'un bug lié aux pluies d'étoiles filantes ayant un interval de temps se terminant à la derniere minute de l'heure (X:59).
+hLog20200630	"MeteoNook has been rewritten from scratch to be more robust and updatable, and is now open-source. There is a new Overview page that shows you information about your upcoming weather and lets you store island information. The seed finder has been made easier to use by adding more 'not sure' options and it can now tolerate small errors in shooting star timing. Dates will now show in M/D/Y format if you select ""English (US)""."	"MeteoNook ha sido reescrito para hacerlo mas robusto y mas fácil de actualizar, y ahora tiene su código abierto. Hay una nueva pestaña, 'Ahora', que muestra información sobre el tiempo actual y que permite guardar los datos de tu isla. El buscador de semillas es más fácil de usar porque tiene mas opciones del tipo 'no estoy seguro' y ahora soporta pequeños fallos en el horario de meteoritos. Además, el idioma ""English (US)"" ahora muestra las fechas en el formato mes/día/año."	"MétéoNook a été entièrement réécrit afin d'être plus robuste et facile à améliorer, et est désormais open-source. Il y a une nouvelle page pour l'Aperçu qui montre les prédiction météo et permet de stocker les informations de l'île. La recherche de graine est désormais plus facile grâce à l'ajout de nouvelles options 'incertains' et tolère des erreurs mineures avec les temps d'apparitions d'étoiles filantes. Les dates sont désormais au format MM/JJ/YYYY si la langue ""English (US)"" est selectionnée."

```

Comment:
```suggestion
hLog20200630	MeteoNook has been rewritten from scratch to be more robust and updatable, and is now open-source. There is a new Overview page that shows you information about your upcoming weather and lets you store island information. The seed finder has been made easier to use by adding more 'not sure' options and it can now tolerate small errors in shooting star timing. Dates will now show in M/D/Y format if you select "English (US)".	MeteoNook ha sido reescrito para hacerlo mas robusto y mas fácil de actualizar, y ahora tiene su código abierto. Hay una nueva pestaña, 'Ahora', que muestra información sobre el tiempo actual y que permite guardar los datos de tu isla. El buscador de semillas es más fácil de usar porque tiene mas opciones del tipo 'no estoy seguro' y ahora soporta pequeños fallos en el horario de meteoritos. Además, el idioma "English (US)" ahora muestra las fechas en el formato mes/día/año.	MétéoNook a été entièrement réécrit afin d'être plus robuste et facile à améliorer, et est désormais open-source. Il y a une nouvelle page pour l'Aperçu qui montre les prédiction météo et permet de stocker les informations de l'île. La recherche de graine est désormais plus facile grâce à l'ajout de nouvelles options 'incertains' et tolère des erreurs mineures avec les temps d'apparitions d'étoiles filantes. Les dates sont désormais au format MM/JJ/YYYY si la langue "English (US)" est selectionnée.
```